eta-company-calendar-backend/src/main/resources/application.yaml

```diff
@@ -13,19 +13,40 @@ spring:
     hibernate:
       ddl-auto: create-drop
 
-      #
-      # MailGun API key
-      #
-      #mailgun.api.base.url = https://api.mailgun.net/v3/sandbox328746234ashdsad3287ajahdgsj234234.mailgun.org
-      #mailgun.api.messages.url = https://api.mailgun.net/v3/sandbox328746234ashdsad3287ajahdgsj234234.mailgun.org/messages
-      #mailgun.api.username = api
-      #mailgun.api.password = key-gfd234be65dasd1e46b57sd25bd9a122
-mailgun:
-    api:
-      base:
-        url: https://api.mailgun.net/v3/sandbox80f7417598574294bc4b07da93863028.mailgun.org
-      messages:
-        url: https://api.mailgun.net/v3/sandbox80f7417598574294bc4b07da93863028.mailgun.org/messages
-      username: api
-      password: key-04a1bda485a9f3627bf27a871d0250f9-52b6835e-8d60dba0
+mailing:
+  mailgun:
+    apiKey: "key-2d5a8931915c81640f655ba3b11bc3b8"
```

Comment:
Valahogy meg kellene oldani, hogy környezeti változóból kapjon értéket. @karolyp van ötleted rá?

Reply:
Akár. Vagy valamilyen parameter store jellegű dologból is jöhetne. Bár gondolom Docker konténerben fog futni, szóval simán jöhet környezeti változóból (akár IDE Run Configjában megadva lokál fejlesztéskor).

Reply:
Oké, csak valamiért nem sikerült `${ENV_NAME}` formában kiolvastatnunk vele tegnap. Bár most megpróbáltam én is és összejött szóval rendben lesz így.